visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts

```diff
@@ -10,11 +10,9 @@ import { StoreService } from "../../state/store.service"
 import { setMarkedPackages } from "../../state/store/fileSettings/markedPackages/markedPackages.actions"
 import { BlacklistType, MarkedPackage, NodeType } from "../../codeCharta.model"
 import { addBlacklistItem } from "../../state/store/fileSettings/blacklist/blacklist.actions"
-import { FocusedNodePathService } from "../../state/store/dynamicSettings/focusedNodePath/focusedNodePath.service"
-import { BlacklistService } from "../../state/store/fileSettings/blacklist/blacklist.service"
-import { MarkedPackagesService } from "../../state/store/fileSettings/markedPackages/markedPackages.service"
 import { focusNode, unfocusNode } from "../../state/store/dynamicSettings/focusedNodePath/focusedNodePath.actions"
 import { NodeDecorator } from "../../util/nodeDecorator"
+import {CodeMapMouseEventService} from "../codeMap/codeMap.mouseEvent.service";
```

Comment:
Nit: maybe run the formatter (prettier) once? :-) We should likely run the formatter automatically. It's placed in the root scope of the monorepo right now.